xma/layers/linear_attention/triton_implementation/__init__.py

```
@@ -19,16 +19,17 @@ def forward(
         v: torch.Tensor,
         h0: torch.Tensor | None,
         attention_multiplier: float,
+        output_state: bool,
         cu_seqlens: torch.Tensor | None,
         max_seqlen: int | None,
-    ) -> tuple[torch.Tensor, torch.Tensor]:
+    ) -> tuple[torch.Tensor, torch.Tensor | None]:
         Nq, Nk, Nv, N = _get_num_heads(q=q, k=k, v=v, run_check=False)
```

Ruff (0.16.1)

[warning] Unpacked variable `Nq` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)